vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerDetachAttachPageIT.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker;
+
+import com.vaadin.flow.testutil.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Integration tests for attaching / detaching date time picker.
+ */
+@TestPath("date-time-picker-detach-attach")
+public class DateTimePickerDetachAttachPageIT extends AbstractComponentIT {
+
+    @Test
+    public void clientSideValidationIsOverriddenOnAttach() {
+        open();
+
+        assertDateTimePickerIsValidOnTab();
+
+        // Detaching and attaching date time picker
+        WebElement toggleAttach = findElement(By.id("toggle-attached"));
```

Comment:
Prefer using `TestBenchElement` (or more specific class like `DateTimePickerElement`) in this file too instead of `WebElement`. Afaik `WebElement` is only used in some older tests, new tests should prefer TestBench based elements like `DateTimePickerElement` or the more generic `TestBenchElement` if needed.

This line can be replaced with:
```java
TestBenchElement toggleAttach = $("button").id("toggle-attached");
```

Reply:
Was aligned with other components.
Don't have strong opinion, updated.